assets/vsphere_features_config.yaml

```diff
@@ -5,6 +5,8 @@ data:
   "online-volume-extend": "true"
   "block-volume-snapshot": "true"
   "improved-volume-topology": "false" # operator will set this value to true if cluster supports CSI topology
+  "improved-csi-idempotency": "true"
+  "block-volume-snapshot": "true"
```

Comment:
Could you add comments to these flags as you add them?

Reply:
As in what they do?

Reply:
done.